src/CommunityToolkit.Maui.Core/Interfaces/IExpander.cs

```diff
@@ -0,0 +1,19 @@
+namespace CommunityToolkit.Maui.Core;
+
+/// <summary>
+/// Allows collapse and expand content.
+/// </summary>
+public interface IExpander : IView
+{
+	public IView Header { get; }
+	public IView Content { get; }
+	public bool IsExpanded { get; }
```

Comment:
This property can be changed from the platform side (user interaction) so we should add the setter.
```suggestion
	public bool IsExpanded { get; set; }
```